src/crawlers.js

```diff
@@ -1,105 +1,187 @@
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(crawlerId, 'crawlerId', 'String');
+
+        return requestPromise({
+            url: `${options.baseUrl}${BASE_PATH}/${userId}/crawlers/${crawlerId}`,
+            json: true,
+            method: 'DELETE',
+            qs: { token },
+        });
+    },
+
+    startExecution: (requestPromise, options) => {
+        const { crawlerId, userId, token, settings } = options;
+
+        checkParamOrThrow(userId, 'userId', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+        checkParamOrThrow(crawlerId, 'crawlerId', 'String');
 
-        const bodyAttributes = ['customId',
-            '_id',
-            'comments',
-            'startUrls',
-            'crawlPurls',
-            'pageFunction',
-            'clickableElementsSelector',
-            'interceptRequest',
-            'considerUrlFragment',
-            'loadImages',
-            'loadCss',
-            'injectJQuery',
-            'injectUnderscoreJs',
-            'ignoreRobotsTxt',
-            'skipLoadingFrames',
-            'verboseLog',
-            'disableWebSecurity',
-            'maxCrawledPages',
-            'maxOutputPages',
-            'maxCrawlDepth',
-            'timeout',
-            'resourceTimeout',
-            'pageLoadTimeout',
-            'pageFunctionTimeout',
-            'maxInfiniteScrollHeight',
-            'randomWaitBetweenRequests',
-            'maxCrawledPagesPerSlave',
-            'maxParallelRequests',
-            'customHttpHeaders',
-            'customProxies',
-            'cookies',
-            'cookiesPersistence',
-            'customData',
-            'finishWebhookUrl',
-        ];
-
-        const body = _.pick(options, bodyAttributes);
         const queryString = _.pick(options, 'token', 'tag', 'wait');
 
         const requestParams = {
             json: true,
             method: 'POST',
-            url: `${options.baseUrl}${BASE_PATH}/${options.userId}/crawlers/${options.crawler}/execute`,
+            url: `${options.baseUrl}${BASE_PATH}/${userId}/crawlers/${crawlerId}/execute`,
             qs: queryString,
         };
-        if (!_.isEmpty(body)) {
-            requestParams.body = body;
+        if (!_.isEmpty(settings)) {
+            requestParams.body = settings;
         }
 
         return requestPromise(requestParams);
     },
 
+    stopExecution: (requestPromise, options) => {
+        const { executionId, token } = options;
+
+        checkParamOrThrow(executionId, 'executionId', 'String');
+        checkParamOrThrow(token, 'token', 'String');
+
+        const requestParams = {
+            json: true,
+            method: 'POST',
+            url: `${options.baseUrl}${BASE_PATH}/execs/${executionId}/stop`,
+            qs: { token },
+        };
+
+        return requestPromise(requestParams);
+    },
+
+    getListOfExecutions: (requestPromise, options) => {
```

Comment:
Same as `getListOfCrawlers` - we should encapsulate result in `{ items: [...] }`